ext/drreg/drreg.c

```diff
@@ -173,15 +184,58 @@ get_where_app_pc(instr_t *where)
  * SPILLING AND RESTORING
  */
 
+static ptr_uint_t drreg_note_base = 0;
+
+/* Get note value for a drreg instr note. */
+static ptr_uint_t
+get_drreg_note_val(uint val)
+{
+    ASSERT(drreg_note_base > 0, "drreg_note_base not initialized");
```

Comment:
Again, use symbolic constants: you have no guarantee that 0 is not a valid value.  Also, this is called many, many times: I would move such an assert to a higher-level less-frequently-called function that calls this, and mark this as inline.

Reply:
Removed it.

Reply:
Marked it as inline too.